ray-operator/controllers/ray/rayservice_controller_test.go

```diff
@@ -234,6 +233,91 @@ var _ = Context("RayService env tests", func() {
 		})
 	})
 
+	Describe("Autoscaler updates RayCluster should not trigger zero downtime upgrade", Ordered, func() {
+		// (1) Create a RayService custom resource
+		// (2) The RayService controller creates a pending RayCluster
+		// (3) Ray Autoscaler scales up the RayCluster
+		// (4) The RayService controller should not trigger zero downtime upgrade
+		// (5) The serve application becomes ready on the pending RayCluster
+		// (6) The pending RayCluster transitions to become the active RayCluster
+		// (7) Ray Autoscaler scales up the RayCluster again
+		// (8) The RayService controller should not trigger zero downtime upgrade
```

Comment:
Do we still need these comments? Maybe they can be merged into the description of `It`. And then in the future if we want to add more `It` in the middle, we don't need to change a lot of numbers in the comment here.

Reply:
updated

Reply:
I made the comment more generic.